index.html

```diff
@@ -64,6 +62,39 @@ <h2>
     <p>
       TBW
     </p>
+    <p id="credential-type-examples">
+      There are many types of digital credential that can be conveyed using
+      this API. Some of these types include:
+    </p>
+    <ul>
+      <li>
+        a driver's license issued by a testing facility
```

Comment:
Maybe just, but also (for those folks who don't drive):

```suggestion
        a driver's license or proof of age card
```

Reply:
```suggestion
        a driver's license or identity card issued by a government
```

If we want to talk about "proof of age", we should probably break it out into a separate bullet item. Examples include [UK PASS scheme](https://www.pass-scheme.org.uk/) and [TruAge](https://www.mytruage.org/).

Reply:
@msporny, your suggestion of "issued by a government" wfm.